presto-main/src/test/java/com/facebook/presto/execution/scheduler/TestConsistentHashingNodeProvider.java

```diff
@@ -56,11 +57,11 @@ public void testDistribution()
     public void testMultipleCandidates()
     {
         List<InternalNode> nodes = IntStream.range(0, 10).mapToObj(i -> new InternalNode(format("other%d", i), URI.create(format("http://127.0.0.%d:100", i)), NodeVersion.UNKNOWN, false)).collect(toImmutableList());
-        ConsistentHashingNodeProvider nodeProvider = ConsistentHashingNodeProvider.create(nodes, 1);
-        assertEquals(ImmutableSet.copyOf(nodeProvider.get("split1", 10)), nodes.stream().map(InternalNode::getHostAndPort).collect(toImmutableSet()));
-        assertEquals(ImmutableSet.copyOf(nodeProvider.get("split1", 11)), nodes.stream().map(InternalNode::getHostAndPort).collect(toImmutableSet()));
+        Optional<ConsistentHashingNodeProvider> nodeProvider = ConsistentHashingNodeProvider.create(nodes, 1);
```

Comment:
We will no longer need changes in this file

Reply:
Done.